src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+function Header() {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '2em',
+  };
+  return (
+    <header style={headerStyle}>
+      <h1 style={{ fontSize: '25px', marginBottom: '15px' }}>
```

Comment:
What if you import the styles from an external Css file, I think it would make your code more easy to read